tests/test_dse.py

```diff
@@ -2711,8 +2711,9 @@ def test_fullopt(self):
         assert np.isclose(summary0[('section0', None)].oi, 2.851, atol=0.001)
 
         assert summary1[('section0', None)].ops == 9
-        assert summary1[('section1', None)].ops == 31
-        assert summary1[('section2', None)].ops == 88
+        assert summary1[('section1', None)].ops == 9
```

Comment:
most of these updates are gone with my branch, I wonder whether we should wait for it to land otherwise I will have to update them all once more, but let's see the timings of things

Reply:
I can trivially make the `ceil/floor` update here, not sure which way is better as might gain a bit still for very large number of sources to hoist it.